config/jobs/cert-manager/csi-lib/cert-manager-csi-lib-presubmits.yaml

```diff
@@ -25,21 +51,19 @@ presubmits:
 
   - name: pull-cert-manager-csi-lib-e2e
     decorate: true
-    # TODO: Keep optional to not block other PRs. Change once e2e test
-    # boilerplate code has been merged to main.
-    always_run: false
-    optional: true
+    always_run: true
     labels:
+      preset-go-cache: "true"
+      preset-local-cache: "true"
       preset-dind-enabled: "true"
     spec:
       containers:
-      - image: europe-west1-docker.pkg.dev/cert-manager-tests-trusted/cert-manager-infra-images/nix-dind:20250327-a3af8ba-2.11.0
```

Comment:
This will allow us to remove our nix image, which was only used here (this was never fully finished).